cylc/rose/stem.py

```diff
@@ -450,10 +450,12 @@ def process(self):
             self.opts.project.append(project)
 
             if i == 0:
-                # Get the name of the template section to be used:
                 template_type = get_rose_vars(
                     Path(url) / "rose-stem")["templating_detected"]
-                self.template_section = f'[{template_type}:suite.rc]'
+                if template_type in ['jinja2', 'empy']:
+                    self.template_section = f'[{template_type}:suite.rc]'
+                else:
+                    self.template_section = f'[{template_type}]'
```

Comment:
Note, this logic is also defined in `identify_templating_section`.

Reply:
Not without changing that logic - at the moment it is only going 
```
if label exists in list:
  return label
```

This does
```
if something within the label exists in list:
  return label
```
I can refactor to use the same code but this is a sig change to Cylc Rose.

Reply:
https://github.com/wxtim/cylc-rose/pull/1 is what I needed to do to work like this...

Reply:
We've had a few issues related to these three sections so I think it would be worth the effort of centralising them for piece of mind.

Reply:
Done.